collections/src/main/scala/strawman/collection/mutable/BitSet.scala

```diff
@@ -85,8 +85,6 @@ class BitSet(protected[collection] final var elems: Array[Long])
 
   def get(elem: Int): Option[Int] = if (contains(elem)) Some(elem) else None
 
-  def unconstrained: collection.Set[Int] = this
```

Comment:
I removed this method that doesn’t exist in the old collection and for which we already have `unsorted`.